dns/name.py

```diff
@@ -35,7 +35,7 @@
 try:
     maxint = sys.maxint
 except:
-    maxint = (1 << (8 * struct.calcsize("P"))) / 2 - 1
+    maxint = (1 << (8 * struct.calcsize("P"))) // 2 - 1
```

Comment:
IMO maxint is integer, thus result should be integer not float

in this way it is compatible with py2:

In [1]: 5/2
Out[1]: 2

In [2]: 5/2.0
Out[2]: 2.5

Because there is no float in expression, py2 will return 2, what is the
same as 5//2 in py3 (and py2)



2016-06-29 9:04 GMT+02:00 Sebastian <notifications@github.com>:

> In dns/name.py
> <https://github.com/rthalley/dnspython/pull/178#discussion_r68894963>:
>
> > @@ -35,7 +35,7 @@
> >  try:
> >      maxint = sys.maxint
> >  except:
> > -    maxint = (1 << (8 * struct.calcsize("P"))) / 2 - 1
> > +    maxint = (1 << (8 * struct.calcsize("P"))) // 2 - 1
>
> // is the integer division operator:
>
> Here an example (python 3):
>
> >>> 5/22.5>>> 5//22
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rthalley/dnspython/pull/178/files/d90346c6ce43a48e9540fa16d60377cc852586aa#r68894963>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AECrYTxLUcwiWkkkdLqz7g5-dgsAKzUYks5qQhkZgaJpZM4I-oYY>
> .
>



-- 
S pozdravom Martin Bašti.